### ISSUE TITLE
A Qiskit Textbook module page has a bug that overloads the Chrome

### ISSUE THREAD

**Issue by dvrshil** [Qiskit/platypus#1654]
### Informations

- **Qiskit version**: N/A
- **Python version**: N/A
- **Operating system**: Mac OS, Chrome Browser

### What is the current behavior?

The site does not load and have to force restart the Chrome (or Safari) each time I try to access the webpage.

### Steps to reproduce the problem

Open the link: https://learn.qiskit.org/course/machine-learning/training-quantum-circuits

### What is the expected behavior?

Browser Overload

### Suggested solutions

Solving some bug I am not sure exists where in the Qiskit web page server.

**Comment by wadleigh** [comment]
I also have an issue with this page in Chrome on both Windows 10 and iPadOS.  If I wait long enough, the page eventually loads, and I see that there is a warning message after the code blocks in the "Natural gradients" section.  The message repeats may times (~50 for the first code block and probably more for the second code block).  It's long enough that scrolling through the whole thing is difficult and I would guess that's what's tying up the browser. 

The warning message is:

"/Users/frankharkins/Library/Python/3.8/lib/python/site-packages/sklearn/linear_model/_base.py:148: FutureWarning: 'normalize' was deprecated in version 1.0 and will be removed in 1.2. Please leave the normalize parameter to its default value to silence this warning. The default behavior of this estimator is to not do any normalization. If normalization is needed please use sklearn.preprocessing.StandardScaler instead.
  warnings.warn("

<img width="738" alt="Natural gradients error 2" src="https://user-images.githubusercontent.com/72709116/200138512-43201d88-3cff-41a2-9e86-33e1164a36ee.png">
<img width="739" alt="Natural gradients error" src="https://user-images.githubusercontent.com/72709116/200138513-644dd802-08a2-45ce-81df-06424e3c2c17.png">
